zulipterminal/themes/zt_dark.py

```diff
@@ -0,0 +1,60 @@
+# ZT DARK __________________________________________________________
+# __________________________________________________________________
+# Using Default Color scheme
+from zulipterminal.themes._template import Color
```

Comment:
It would be nice if all colors were defined i the same way.
But in this method, `DefaultColor -> ColorWithBold -> from file import ColorWithBold as Color`
But if another file wants to use gruvbox colors, `GruvBoxColor -> Color -> from file import Color`
If everything was similar to template everything is easily understood.
Hence also REQUIRED_STYLES makes sense. `_template` would be main file to understand how themefiles should look.

Reply:
Right, but a theme file doesn't have a `REQUIRED_STYLES` dict in it, so having a "template" which has it in feels confusing.

Reply:
I'll close this and we could discuss this above.